src/scene-entry-manager.js

```diff
@@ -142,9 +142,9 @@ export default class SceneEntryManager {
 
   _setupPlayerRig = () => {
     this._setPlayerInfoFromProfile();
+    this.store.addEventListener("statechanged", this._setPlayerInfoFromProfile);
```

Comment:
This is a little dangerous, since it could result in an infinite loop if you have a 404'd avatar in local storage. `_setPlayerInfoFromProfile` will call `resetToRandomLegacyAvatar`, which updates store state.
This will be fixed in my PR anyway #1355

Reply:
This was already being done before, I just moved it so that it didn't get added multiple times as it is now in production